ios/Plugin/PluginHelperExtensions.swift

```diff
@@ -41,6 +41,29 @@ internal extension PurchasesPlugin {
         return handleResponse
     }
 
+    func getCompletionBlockHandlerForArrayResponse(
```

Comment:
The function signature for fetching eligible winback offers is:

```swift
static func eligibleWinBackOffers(
     for productIdentifier: String,
     completion: @escaping ([[String: Any]]?, ErrorContainer?) -> Void
 )
```

Note the `[[String: Any]]?` in the callback - it returns an array of objects. This function allows us to parse that response type, since the normal `getCompletionBlockHandler` assumes a return type of `[String: Any]?`